env0/provider.go

```diff
@@ -127,7 +128,13 @@ func configureProvider(version string, p *schema.Provider) schema.ConfigureConte
 		if err != nil {
 			return nil, diag.Diagnostics{diag.Diagnostic{Severity: diag.Error, Summary: err.Error()}}
 		}
+		apiClient := client.NewApiClient(httpClient)
+		if apiClient == nil {
```

Comment:
Not required. apiClient will never be nil.

```
func NewApiClient(client http.HttpClientInterface) ApiClientInterface {
	return &ApiClient{
		http:                 client,
		cachedOrganizationId: "",
	}
}

```

Reply:
[e137d6d](https://github.com/env0/terraform-provider-env0/pull/456/commits/e137d6d4ba10183216f6e9d2e64f5246732fcd2f)